sites/partners/src/components/listings/PaperListingForm/sections/BuildingDetails.tsx

```diff
@@ -206,7 +207,6 @@ const BuildingDetails = ({
             }}
             register={register}
           />
-          <p className="field-sub-note">{t("listings.requiredToPublish")}</p>
```

Comment:
is this cruft?

Reply:
No. This was moved to line 297 to be below all the address fields.
Before: 
![image](https://user-images.githubusercontent.com/12400715/234313295-b4e99846-882c-4c34-90e0-b1678d4e3636.png)

After:
![image](https://user-images.githubusercontent.com/12400715/234314325-95184366-f998-46b2-8e11-0ede3bb2189a.png)

Reply:
gotcha, thanks